Formula/node.rb

```diff
@@ -32,8 +32,8 @@ class Node < Formula
   # We track major/minor from upstream Node releases.
   # We will accept *important* npm patch releases when necessary.
   resource "npm" do
-    url "https://registry.npmjs.org/npm/-/npm-7.18.1.tgz"
-    sha256 "afc22878cdd5d63b298d62abe832ee7fe37795cd8ccb1fe689ad31d28c790b08"
+    url "https://registry.npmjs.org/npm/-/npm-7.19.0.tgz"
```

Comment:
Is this the versioned bundled with Node 16.4.1? I don't see it in the changelog.

Reply:
Upstream did an upgrade in [this PR](https://github.com/nodejs/node/pull/39148) and there is [another one](https://github.com/nodejs/node/pull/39148) going on.

I checked the artifact, and found the npm version is with `7.18.1`

Reply:
What would be the preferred way proceeding in here.

Reply:
The bump to 7.19.0 is in this commit: https://github.com/nodejs/node/commit/9c85be1278bac42df701c50becca5eee08b39466 This isn't in the Node 16.4.1 release tag.

Since we use this resource to track the version that comes with Node, per

https://github.com/Homebrew/homebrew-core/blob/aecea0d0fb929fbdfb6476f481fb852584557ab5/Formula/node.rb#L32

I think we should revert the change to the `resource`.

Reply:
I've removed @chenrui333's commit from the branch.